numalogic/connectors/druid.py

```diff
@@ -100,14 +104,21 @@ def fetch(
         dimensions: list[str],
         granularity: str = "minute",
         aggregations: Optional[dict] = None,
+        post_aggregations: Optional[dict] = None,
```

Comment:
This comment is off the main topic: build_params() debug-logs the query params, while this fetcher init() debug-logs the query latency. Can we combine both debug logs into one log?

When logs from many vertices / pods are ingested into log store (such as Splunk), two locally back-to-back log lines may be interleaved by other log lines, making it harder to correlated separate logs.

Reply:
I've moved them into a single log